dials.go

```diff
@@ -56,7 +56,7 @@ func Config(ctx context.Context, t interface{}, sources ...Source) (*View, error
 		return nil, err
 	}
 
-	view := &View{
+	view := &Dials{
```

Comment:
nit: let's rename `view` on the left side to dials too and carry forward through the rest of the function

Reply:
Heh -- I thought I had found all of those .. thanks!